scipy/spatial/transform/rotation.py

```diff
@@ -1549,6 +1633,31 @@ def __getitem__(self, indexer):
         """
         return self.__class__(self._quat[indexer], normalized=True)
 
+    @classmethod
+    def identity(cls, num=None):
+        """Get identity rotation(s).
+
+        The identity rotation is its own inverse. Composition of a rotation
+        with the identity rotation leaves the rotation unchanged.
+
+        Parameters
+        ----------
+        num : int or None, optional
+            Number of identity rotations to generate. If None (default), then a
+            single rotation is generated.
+
+        Returns
+        -------
+        identity : Rotation object
+            The identity rotation.
+        """
+        if num is None:
+            q = [0, 0, 0, 1]
+        else:
+            q = np.zeros((num, 4))
+            q[:, 3] = 1
+        return Rotation.from_quat(q)
```

Comment:
self.__class__(q, normalized=True, copy=False) 

(We better change normalized to normalize, it's easier to perceive this way).

Reply:
There is no `self` since it is a class method.  Should it be `cls.from_quat(q)`?

The `random` method (also a class method) uses: `return Rotation.from_quat(sample)`
Should I change that too?

Reply:
Of course `cls`. 

I believe in random the normalization is needed, that's why `from_quat` is used.

Reply:
OK.  I have changed both to use `cls` for consistency.  Default `cls` initialization performs normalization.